pkgs/development/python-modules/pysnooper/default.nix

```
@@ -0,0 +1,33 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, six
+, future
+, decorator
+}:
+
+buildPythonPackage rec {
+  version = "0.0.21";
+  pname = "PySnooper";
```

```suggestion
  pname = "pysnooper";
```
You'll have to specify it again within `fetchPypi`.